File: dbflow-tests/src/androidTest/java/com/raizlabs/android/dbflow/test/sql/TestMigration.java
package com.raizlabs.android.dbflow.test.sql;

import com.raizlabs.android.dbflow.sql.migration.Migration;
import com.raizlabs.android.dbflow.structure.database.DatabaseWrapper;
import com.raizlabs.android.dbflow.test.TestDatabase;

@com.raizlabs.android.dbflow.annotation.Migration(version = 1, database = TestDatabase.class, priority = 1)
public class TestMigration implements Migration {
    @Override
    public void onPreMigrate() {

    }

    @Override
    public void migrate(DatabaseWrapper database) {

    }

    @Override
    public void onPostMigrate() {

    }
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/sql/TestMigration.kt
package com.raizlabs.android.dbflow.sql

import com.raizlabs.android.dbflow.sql.migration.AlterTableMigration
import com.raizlabs.android.dbflow.TestDatabase
import com.raizlabs.android.dbflow.structure.TestModel1

@com.raizlabs.android.dbflow.annotation.Migration(version = 1, database = TestDatabase::class, priority = 1)
class TestMigration(table: Class<TestModel1>) : AlterTableMigration<TestModel1>(table)
